R/simfix.R

```diff
@@ -40,6 +41,7 @@ NULL
 #' @param rg As in `simtrial::tenFHCorr()`.
 #' A \code{tibble} with variables \code{rho} and \code{gamma}, both greater than equal
 #' to zero, to specify one Fleming-Harrington weighted logrank test per row.
+#' @param setSeed Bool: Set seed for simulation?
```

Comment:
One usually wants to set seed for reproducibility. I guess it would be a more meaningful argument if this is an integer input so that people can **control** what (initial) seed is used.